File: exs/mundo_1/c#/027.cs
/*
Desafio 027

Problema: Faça um programa que leia o nome completo de uma
          pessoa, mostrando em seguida o primeiro e o último
          nome separadamente.

Resolução do problema:
*/
using System;

class io {
    public static string input(string text) {
        Console.Write(text);
        return Console.ReadLine();
    }
    
    public static void print(object data, string end="\n") {
        string text = data.ToString();

        Console.Write($"{text}{end}");
    }
}

class Program {
  public static void Main () {
    string nome = io.input("Digite seu nome completo: ");

    string[] listaNome = nome.Split(' ');

    string primeiroNome = listaNome[0];
    string ultimoNome = listaNome[listaNome.Length - 1];

    io.print($"Primeiro nome: {primeiroNome}");
    io.print($"Ultimo nome: {ultimoNome}");
  }
}

File: exs/mundo_1/java/027.java
/*
    ### 027 - Primeiro e último nome de uma pessoa
    Faça um programa que leia o nome completo de uma pessoa, mostrando em seguida o primeiro e o último nome separadamente
 */


import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        String nome;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Digite o nome completo de uma pessoa: ");
        nome = scanner.nextLine();

        String [] nomeSplitado = nome.split(" ");
        System.out.println("Primeiro nome: " + nomeSplitado[0]);
        System.out.println("Ultimo nome: " + nomeSplitado[nomeSplitado.length - 1]);

    }

}
